src/libc-wrappers/libc-wrappers.c

```diff
@@ -16,8 +16,145 @@
 
 
 #include <signal.h>
+#include <pthread.h>
+#include <link.h>
 
 
+/*
+  __libc_start_main wrapping
+ */
+
+struct startup_info
+{
+	void *sda_base;
```

Comment:
Nitpick: would be good to avoid tabs in the C sources.

(We use tabs in the Go sources because that's what `go fmt` mandates.)

Reply:
I took the liberty to update the branch.